### DISCUSSION TITLE
can't find git path after updating macOS

### DISCUSSION THREAD

**Discussion by neemazephania** [IBAHCM/RPiR#194] [Archive] [answered]
I've just updated macOS  but i'm getting "git was not detected on the system path" error on R studio, and yet there is default git path and git is already installed.  I tried to change path manually but it wasn't a success. 

also, I've tried couple of online solutions like installing Xcode command line tools, updating home-brew but none of that are working.

can you help me? @richardreeve @soniamitchell

**Comment by soniamitchell** [comment, marked as answer]
Hey @neemazephania, have you tried this? https://happygitwithr.com/rstudio-see-git.html

**Reply by neemazephania** [reply]
Thanks @soniamitchell, found it in "/usr/local/bin/git".

**Reply by soniamitchell** [reply]
Awesome 👍
